activerecord/lib/active_record/relation/calculations.rb

```diff
@@ -155,7 +155,7 @@ def calculate(operation, column_name)
     end
 
     # Use #pluck as a shortcut to select one or more attributes without
-    # loading a bunch of records just to grab the attributes you want.
+    # loading a bunch of ActiveRecord objects just to grab the attributes you want.
```

Comment:
"Active Record" needs a space.  But, looking at the [`pick` docs](https://github.com/rails/rails/blob/296ef7a17221e81881e38b51aa2b014d7a28bac5/activerecord/lib/active_record/relation/calculations.rb#L220), what do you think about this alternative:

```suggestion
    # loading an entire record object per row.
```

Or even this one:

```suggestion
    # instantiating a model object per row.
```

Reply:
Second one looks perfect. Do you think we can use `Active Record` instead of `model`, because we also have [Active Model](https://guides.rubyonrails.org/active_model_basics.html).

Reply:
Here, "model" refers to the "model" in [model-view-controller](https://en.wikipedia.org/wiki/Model%E2%80%93view%E2%80%93controller), not Active Model.

Conversely, "Active Record" is a framework which defines many objects / classes, `ActiveRecord::Base` among them.

All in all, if you would prefer not to use the word "model", I would go with "entire record object".